src/services/iiif-parser.js

```diff
@@ -186,6 +186,12 @@ export function getMediaInfo({ manifest, canvasIndex, srcIndex = 0 }) {
     });
     // Set default src to auto
     sources = setDefaultSrc(resources, isMultiSource, srcIndex);
+    // If manifest has a start, set canvas sources' time fragments to match
+    let manifestStart;
+    if (manifest.start !== undefined && manifest.start.source === canvas.id) {
+      manifestStart = manifest.start.selector.t;
+      sources = setDefaultStart(sources, manifestStart, duration);
```

Comment:
Is it more feasible to read start property this way than using `getCustomStart()` method in the parser?